.github/ISSUE_TEMPLATE/b_make_a_suggestion.yml

```diff
@@ -0,0 +1,37 @@
+name: Suggestion
+description: Make a suggestion for something that you think should be added to Dink.
+labels:
+  - enhancement
+body:
+  - type: checkboxes
+    id: acknowledgments
+    attributes:
+      label: Checklist
+      description: null
+      options:
+        - label: I've searched the [issues and pull
+            requests](https://github.com/pajlads/Dinkplugin/issues?q=) and the
+            `Unreleased` section of the
+            [changelog](https://github.com/pajlads/DinkPlugin/blob/master/CHANGELOG.md)
+            for similar looking suggestions.
```

Comment:
I want this included, I'm just not sure how good it is since we do dev-style changelogs here versus the user-friendly ones I force into Chatterino

Reply:
I agree with keeping the unreleased part (we've gotten multiple issues in the past that were already fixed)

can we make it a separate checkbox tho? (ppl are too lazy to read the full line)